virttest/libvirt_vm.py

```diff
@@ -2137,6 +2137,46 @@ def detach_interface(self, option="", ignore_status=False,
                                       ignore_status=ignore_status,
                                       debug=debug)
 
+    def attach_device(self, xml_file, options="", ensure_boot=False, debug=False):
```

Comment:
I'm wondering if it'd be better to use `ensure_guest_booted` (same for `detach_device`).